src/auto-evo/RunResults.cs

```diff
@@ -311,6 +323,16 @@ void OutputPopulationForPatch(Species species, Patch patch, long population)
 
                 if (playerReadable)
                     builder.Append("\n");
+
+                if (GetGlobalPopulation(entry.Species) == 0 && !moving)
```

Comment:
Hang on, can you prevent extinction by moving?
I think that the algorithm should also run the population simulation in the target patch, so that if you send population that population might die out anyway...
So I would prefer an approach where the global population is used but the adjustments from moving are also applied, instead of just blankedly assuming that if a species is moving it can't go extinct.

Reply:
Aight